src/app/pages/form/form.component.ts

```diff
@@ -1,9 +1,72 @@
 import { Component } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+
+interface City {
+    name: string;
+  //   code: string;
```

Comment:
to było spoko, odkomentowałabym to zdecydowanie. 
Każde miasto byłoby obiektem `City`. 
Na przykładzie Nowego Yorku który masz poniżej można by to było zrobić tak:
`{name: 'New York, USA',  code: 'JFK'}`
`code` który widzisz powyżej to kod przykładowego lotniska z nowego yorku :)